src/shared/sleep-products.conf

```diff
@@ -33,4 +33,6 @@
 #
 #  Note: All the sections and variables are optional
 
+[CanSuspend]
+BlackListProducts=F3C EE-200 # https://phabricator.endlessm.com/T20298
```

Comment:
Hmmm... I'm not sure whether this "inline" comment will be safely ignored or parsed along with the other product names (it's been a while since I touched this!). Better safe than sorry, I think I'd have put the comment in a separate line

Reply:
I checked the parser and you're right; it only checks the first character.